repos/system_upgrade/el7toel8/actors/checkosrelease/actor.py

```diff
@@ -19,43 +17,9 @@ class CheckOSRelease(Actor):
     name = 'check_os_release'
     consumes = (OSReleaseFacts,)
     produces = (Report,)
-    tags = (ChecksPhaseTag, IPUWorkflowTag,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
 
     def process(self):
-        skip_check = os.getenv('LEAPP_SKIP_CHECK_OS_RELEASE')
-        if skip_check:
-            report_generic(
-                title='Skipped OS release check',
-                summary='OS release check skipped via LEAPP_SKIP_CHECK_OS_RELEASE env var.',
-                severity='low'
-            )
-            return
-
-        min_supported_version = {
-            'rhel': '7.6'
-        }
-
-        for facts in self.consume(OSReleaseFacts):
-            if facts.release_id not in min_supported_version.keys():
-                report_generic(
-                    title='Unsupported OS id',
-                    summary='Supported OS ids for upgrade process: ' + ','.join(min_supported_version.keys()),
-                    flags=['inhibitor']
-                )
-                return
-
-            min_version = [int(x) for x in min_supported_version[facts.release_id].split('.')]
-            os_version = [int(x) for x in facts.version_id.split('.')]
-
-            for current, minimal in zip(os_version, min_version):
-                if current > minimal:
-                    break
-
-                if current < minimal:
-                    report_generic(
-                        title='Unsupported OS version',
-                        summary='Minimal supported OS version for upgrade process: {}'.format(
-                            min_supported_version[facts.release_id]),
-                        flags=['inhibitor']
-                    )
-                    return
+        supported_version = {'rhel': '7.6'}
```

Comment:
I don't think this is what you would want - You'd rather want now the value to be a list as the plan is that other releases might be added
```suggestion
        supported_version = {'rhel': ['7.6']}
```

Reply:
Very good idea! thanks.